book/website/reproducible-research/rdm/rdm-visualisation.md

```diff
@@ -39,16 +49,36 @@ Below follow some resources that may help you in creating clearer, more accessib
 - [Fundamentals of Data Visualisation](https://serialmentor.com/dataviz/) by {cite:ps}`Wilke2019dataviz`
 - [Data Visualisation, A practical introduction](https://socviz.co/) by {cite:ps}`Healy2018dataviz`
 - [Creating clear and informative image-based figures for scientific publications](https://doi.org/10.1371/journal.pbio.3001161) by {cite:ps}`Jambor2021dataviz`
+- [A layered grammar of graphics](http://dx.doi.org/10.1198/jcgs.2009.07098) by {cite:ps}`Wickham2010dataviz`
 
 ### Other text resources and examples
 
 - [Data Visualisation Course](https://exploratory-data-visualization.netlify.app/)
 - [Free visual and communication resources](https://padlet.com/evansemporiumstore/lks3aoyyhkpnkmpe)
 - [A collection of notebooks demonstrating plotting with matplotlib](https://github.com/KirstieJane/NH19-Visualization/)
+- [A Comprehensive Guide to the Grammar of Graphics for Effective Visualization of Multi-dimensional Data](https://towardsdatascience.com/a-comprehensive-guide-to-the-grammar-of-graphics-for-effective-visualization-of-multi-dimensional-1f92b4ed4149)
+- [Friends Don't let Friends...](https://github.com/cxli233/FriendsDontLetFriends)
```

Comment:
The full title is "Friends Don't Let Friends Make Bad Graphs" and they have DOI: [10.5281/zenodo.7097522](https://dx.doi.org/10.5281/zenodo.7097522).

Reply:
Oh very nice - I overlooked that. Guess I now have to add it to the bibliography hehe